### PR TITLE
remove unused variable

### PR DESCRIPTION
**Pull request by ycphs**

### CODE REVIEW

File: R/WorkbookClass.R [hunk 1]
```diff
@@ -17819,7 +17819,8 @@ Workbook$methods(
 
     ## Increment priority of conditional formatting rule
     if (length((worksheets[[sheet]]$conditionalFormatting)) > 0) {
-      for (i in length(worksheets[[sheet]]$conditionalFormatting):1) {
+      for (i in order(seq_along(worksheets[[sheet]]$conditionalFormatting),
```
Comment (JanMarvin) [inline R/WorkbookClass.R]: I'm wondering if this is intended?

File: R/WorkbookClass.R [hunk 2]
```diff
@@ -1994,9 +1994,9 @@ Workbook$methods(
                                xlworksheetsDir,
                                xlworksheetsRelsDir) {
     ## write worksheets
-    nSheets <- length(worksheets)
-
-    for (i in 1:nSheets) {
+    # nSheets <- length(worksheets)
+    
+    for (i in seq_along(worksheets)) {
```
Comment (JanMarvin) [inline R/WorkbookClass.R:2001]: why not `seq_len(nSheets)`?

File: tests/testthat/test-encoding.R [hunk 3]
```diff
@@ -57,9 +57,9 @@ test_that("Support non-ASCII strings not in UTF-8 encodings", {
   wb <- createWorkbook(creator = non_ascii[1])
   ws <- addWorksheet(wb, non_ascii[2])
   writeDataTable(wb, ws, non_ascii_df, tableName = non_ascii[3])
-  writeData(wb, ws, non_ascii_df, xy = list("D", 1), name = non_ascii[4])
+  writeData(wb, ws, non_ascii_df, xy = c(4, 1), name = non_ascii[4])
```
Comment (JanMarvin) [inline tests/testthat/test-encoding.R]: why are these changes included? did something else change and now they bark?

File: tests/testthat/test-writing_sheet_data.R [hunk 4]
```diff
@@ -168,7 +168,7 @@ test_that("Writing sheetData rows XML - iris", {
     "<row r=\"151\"><c r=\"A151\" t=\"n\"><v>5.9</v></c><c r=\"B151\" t=\"n\"><v>3</v></c><c r=\"C151\" t=\"n\"><v>5.1</v></c><c r=\"D151\" t=\"n\"><v>1.8</v></c><c r=\"E151\" t=\"s\"><v>7</v></c></row>"
   )
 
-  for (i in 1:length(expected_rows)) {
+  for (i in seq_along(expected_rows)) {
```
Comment (JanMarvin) [inline tests/testthat/test-writing_sheet_data.R:171]: I personally would have replaced it with `seq_len(length(...))`. yours does look better, but I've been bitten enough not to blindly trust `seq_along` for anything not data.frame related.

File: R/WorkbookClass.R [hunk 5]
```diff
@@ -2633,17 +2634,16 @@ Workbook$methods(
...
         ## now replace
         worksheets[[sheet]]$conditionalFormatting[[i]] <<-
           gsub(priority_pattern,
-            priority_new,
-            worksheets[[sheet]]$conditionalFormatting[[i]],
-            fixed = TRUE
-          )
+               priority_new,
+               worksheets[[sheet]]$conditionalFormatting[[i]],
+               fixed = TRUE)
```
Comment (JanMarvin) [inline R/WorkbookClass.R]: unintended include of this chunk?

**Review by JanMarvin — COMMENTED** [review]
Thanks for taking care of these!

File: R/WorkbookClass.R (hunk 2 above)
Comment (ycphs) [reply R/WorkbookClass.R:2001]: Because it is the equivalent and nSheets is just used for this loop.

Comment (JanMarvin) [reply R/WorkbookClass.R:2001]: oh, yes. sleepy me 👍

File: R/WorkbookClass.R (hunk 1 above)
Comment (JanMarvin) [reply R/WorkbookClass.R]: oh nevermind. missed the `:1` while looking at the pull request this morning. I might have tried `rev()`, but it's fine this way

File: R/WorkbookClass.R (hunk 5 above)
Comment (ycphs) [reply R/WorkbookClass.R]: Yes unintended

File: R/WorkbookClass.R (hunk 1 above)
Comment (ycphs) [reply R/WorkbookClass.R]: changed to rev

File: R/WorkbookClass.R (hunk 5 above)
Comment (ycphs) [reply R/WorkbookClass.R]: removed the changes

File: tests/testthat/test-encoding.R (hunk 3 above)
Comment (ycphs) [reply tests/testthat/test-encoding.R]: Changed back to the initial code

**Comment by ycphs** [comment]
The failed check was code coverage
